Tests/ApolloCodegenTests/ApolloCodegenConfigurationTests.swift

```diff
@@ -41,7 +44,8 @@ class ApolloCodegenConfigurationTests: XCTestCase {
   func test_validation_givenSchemaPath_doesNotExist_shouldThrow() throws {
     // given
     let schemaPath = directoryURL.appendingPathComponent(UUID().uuidString)
-    let config = ApolloCodegenConfiguration(input: .init(schemaPath: schemaPath.path),
+    let config = ApolloCodegenConfiguration(input: .init(schemaPath: schemaPath.path,
+                                                         includes: ["**/*.graphql"]),
```

Comment:
I think this param may need to be renamed from `includes` since it also can give information about what paths need to be excluded as well. 

`searchPaths` is what comes to mind off the top of my head but there's probably a better one

Reply:
This last change was done in https://github.com/apollographql/apollo-ios/pull/2041/commits/afe85ad3850ee76a1def5fa9655a4b6a3d107754.